scripts/clean-sc.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+SCRIPTS_PATH="$(dirname "$(readlink -f "$0")")"
```

Comment:
Should we name this `here` to make it same as the scripts in `bin/`? I also think that would make it easier to understand.
```suggestion
here="$(dirname "$(readlink -f "$0")")"
```